Build/Module.xml

```diff
@@ -1,11 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<ModuleInfo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <Name>MonoGame.Framework</Name>
-  <ModuleAssemblies />
-  <DefaultAction>Generate</DefaultAction>
-  <SupportedPlatforms>Android,Angle,Linux,Ouya,Windows8,Windows,WindowsGL,WindowsPhone,WindowsPhone81,WindowsUAP,iOS,MacOS,Web</SupportedPlatforms>
-  <DefaultWindowsPlatforms>Android,Angle,Linux,Ouya,Windows8,Windows,WindowsGL,WindowsPhone,WindowsPhone81,WindowsUAP,iOS</DefaultWindowsPlatforms>
-  <DefaultMacOSPlatforms>Angle,MacOS,iOS,WindowsGL,Android</DefaultMacOSPlatforms>
-  <DefaultLinuxPlatforms>Angle,Linux,WindowsGL</DefaultLinuxPlatforms>
-  <DisableSynchronisation>true</DisableSynchronisation>
-</ModuleInfo>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Module>
+  <Name>MonoGame.Framework</Name>
+  <DefaultAction>Generate</DefaultAction>
+  <DefaultLinuxPlatforms>Angle,Linux,WindowsGL</DefaultLinuxPlatforms>
+  <DefaultMacOSPlatforms>Angle,MacOS,iOS,WindowsGL,Android</DefaultMacOSPlatforms>
+  <DefaultWindowsPlatforms>Android,Angle,Linux,Ouya,Windows8,Windows,WindowsGL,WindowsPhone,WindowsPhone81,WindowsUAP,iOS</DefaultWindowsPlatforms>
+  <SupportedPlatforms>Android,Angle,Linux,Ouya,Windows8,Windows,WindowsGL,WindowsPhone,WindowsPhone81,WindowsUAP,iOS,MacOS,Web</SupportedPlatforms>
+  <DisableSynchronisation>true</DisableSynchronisation>
+  <GenerateNuGetRepositories>true</GenerateNuGetRepositories>

```

Comment:
This generates the "repositories.config" file if you have a "packages.config" file in any of the projects.  "true" was the default value before.  We switched away from the .NET serializer to just reading / writing XML nodes manually to avoid the serializer saving fields we didn't want when run under Mono.  This means that all of the nodes now get written out to the module configuration file.